rts/Sim/Units/UnitMemPool.h

```diff
@@ -7,11 +7,15 @@
 #include "Sim/Misc/GlobalConstants.h"
 #include "System/MemPoolTypes.h"
 
+/* Size asserted manually in UnitHandler.cpp (to avoid extra #includes here).
+ * TODO: would be good if this could be found automatically via metaprogramming */
+using LargestDerivedFromCUnit = CBuilder;
```

Comment:
This will be the only line that needs changing in the future if a different type becomes larger.